spec/lib/object_serializer_class_methods_spec.rb

```diff
@@ -251,7 +251,7 @@
           attributes :name, :release_year
           has_many :actors
           belongs_to :owner, record_type: :user
-          belongs_to :movie_type, serializer: "#{key_transform}_movie_type".to_sym
+          belongs_to :movie_type, serializer: "#{key_transform}_movie_type"
```

Comment:
Also, i'm not sure what's the reason for this change - it seems to fail the spec 😞